ext/_parser.c

```diff
@@ -314,30 +333,67 @@ static PyMethodDef module_methods[] = {
     {NULL}  /* Sentinel */
 };
 
-#ifndef PyMODINIT_FUNC /* declarations for DLL import/export */
```

Comment:
C ext boilerplate I guess, I don't remember. Probably for windows support looking at the comment, not supported anyway.